CHANGES.md

```diff
@@ -11,6 +11,11 @@
 * Adding an end date to `CMIP6_UofT`'s temporal extent for better rendering in STAC Browser
 * Updates to datacube extension helper routines for `CMIP6_UofT`.
 * Make pyessv-archive a requirement for *only* the cmip6 implementation instead of for the whole CLI
+* Fix bug where logger setup failed
+* Simplify CLI argument constructor code (for cleaner and more testable code)
+* Add tests for CLI and implementations when invoked through the CLI
+* Refactored code dealing with requests and authentication to the `requests.py` file
```

Comment:
Add the `STACpopulator` prefix. Too easy to confuse `requests.py` with the "Python `requests` package".